test/test.googleauth.ts

```diff
@@ -1221,6 +1264,10 @@ describe('googleauth', () => {
   it('should make the request', async () => {
     const url = 'http://example.com';
     const {auth, scopes} = mockGCE();
+
+    // tslint:disable-next-line no-any
+    sinon.stub(auth as any, 'getDefaultServiceProjectId').resolves();
```

Comment:
This should not be required.  This is exactly what `createGetProjectIdNock` is supposed to do.

Reply:
Changed - please take a look!